staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -67,11 +67,43 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 				obj.Default = `{"some": {"json": "test"}, "string": 42}` // some valid json
 			}
 			if c.RandBool() {
-				obj.Enum = []interface{}{c.Uint64(), c.RandString(), c.RandBool()}
+				obj.Enum = []interface{}{c.Float64(), c.RandString(), c.RandBool()}
 			}
 			if c.RandBool() {
 				obj.Example = "foobarbaz"
 			}
 		},
+		func(obj *apiextensions.JSONSchemaPropsOrBool, c fuzz.Continue) {
+			if c.RandBool() {
+				c.Fuzz(obj.Schema)
+			} else {
+				obj.Allows = c.RandBool()
+			}
+		},
+		func(obj *apiextensions.JSONSchemaPropsOrArray, c fuzz.Continue) {
+			// disallow both Schema and JSONSchemas to be nil.
+			c.Fuzz(obj.Schema)
+			if obj.Schema == nil {
+				c.Fuzz(&obj.JSONSchemas)
+				if len(obj.JSONSchemas) == 0 {
```

Comment:
you can also create the array yourself and then call c.Fuzz on each element. Would maybe be slightly cleaner.